storage_test.go

```diff
@@ -0,0 +1,11 @@
+package tprstorage
+
+import (
+	"testing"
+
+	"github.com/giantswarm/microstorage"
+)
+
+func TestInterface(t *testing.T) {
```

Comment:
I'll try to experiment with test framework etcd-operator has. I discussed with @JosephSalisbury how to integrate it into archtect https://gigantic.slack.com/archives/C3C7ZQXC1/p1501598136111860